addons/account/models/account_payment.py

```diff
@@ -968,10 +968,16 @@ def unmark_as_sent(self):
 
     def action_post(self):
         ''' draft -> posted '''
-        # Do not allow to post if the account is required but not trusted
-        for payment in self:
-            if payment.require_partner_bank_account and not payment.partner_bank_id.allow_out_payment:
-                raise UserError(_('To record payments with %s, the recipient bank account must be manually validated. You should go on the partner bank account in order to validate it.', self.payment_method_line_id.name))
+        # Do not allow posting if the account is required but not trusted
+        invalid_partner_banks = self.filtered(
+            lambda payment: payment.require_partner_bank_account and not payment.partner_bank_id._is_trusted()
```

Comment:
is_trusted() wasn't used before, probably a mistake since the previously existing comment was already explicit about it... But you should mention it in the commit message as well

Reply:
Sorry, copilot junk, fixed.